pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -840,8 +845,7 @@ func (m *kubeGenericRuntimeManager) removeContainerLog(containerID string) error
 		return fmt.Errorf("failed to get container status %q: %v", containerID, err)
 	}
 	labeledInfo := getContainerInfoFromLabels(status.Labels)
-	annotatedInfo := getContainerInfoFromAnnotations(status.Annotations)
-	path := buildFullContainerLogsPath(labeledInfo.PodUID, labeledInfo.ContainerName, annotatedInfo.RestartCount)
+	path := BuildContainerLogsDirectory(labeledInfo.PodUID, labeledInfo.ContainerName)
```

Comment:
Sorry... This is my fault... We can't remove the whole directory... There might be newer instance of container log with different restart count in this directory. Let's leave the container log directory to be deleted when pod deletion.

Reply:
yep :) Cool will update